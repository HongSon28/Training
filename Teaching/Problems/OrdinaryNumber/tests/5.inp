203
182521
320580
357731
568076
280567
938433
689148
964837
246532
584297
650376
962193
579282
821560
295026
557770
856892
361150
523699
71571
185271
433009
892976
871763
599395
573341
749167
166491
509367
229291
504065
51377
334089
529777
125369
607057
284016
983555
720581
640727
221691
364431
853070
239091
199266
200997
295041
621989
825793
353526
741020
638738
532801
602477
852849
521508
318191
468134
75431
459981
13449
661576
109509
170788
713508
321441
854425
911235
415649
879669
531797
735317
241951
14521
215937
680215
511682
4734
244095
929727
687392
401971
464095
829585
614069
152951
584545
20401
13083
597819
610765
555461
296001
784677
198049
869341
295840
547559
879884
487465
454751
170788
687141
184029
140529
601639
635423
21776
421367
766538
130233
230832
27551
368679
945963
993069
657897
766650
742719
728173
479384
90483
441025
235236
912626
643082
102475
282143
83316
333621
460361
532587
918
260994
535750
859903
919121
888681
413310
176674
740640
843266
272929
553369
431967
856065
237892
109330
876157
15326
581977
367001
439012
104563
419369
980516
550153
78215
139649
785451
488051
551582
779757
95694
75370
762658
992335
833089
140168
245228
136483
712056
818961
258301
84916
680065
298253
189415
337009
450497
543615
948609
137331
220856
253166
838791
440187
22203
521990
707026
688289
73294
610489
656091
842757
321442
466898
396496
258641
27391
664119
907173
51476
