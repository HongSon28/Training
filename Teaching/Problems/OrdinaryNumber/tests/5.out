46
47
48
50
47
53
51
53
47
50
50
53
50
52
47
50
52
48
49
42
46
48
53
52
50
50
51
46
49
47
49
40
48
49
46
50
47
53
51
50
46
48
52
47
46
46
47
50
52
48
51
50
49
50
52
49
47
49
42
49
37
50
45
46
51
47
52
53
48
52
49
51
47
37
46
51
49
31
47
53
51
48
49
52
50
46
50
37
37
50
50
49
47
52
46
52
47
49
52
49
49
46
51
46
46
50
50
37
48
51
46
47
38
48
53
53
50
51
51
51
49
44
48
47
53
50
45
47
43
48
49
49
26
47
49
52
53
52
48
46
51
52
47
49
48
52
47
45
52
37
50
48
48
45
48
53
49
43
46
52
49
49
52
44
42
51
53
52
46
47
46
51
52
47
43
51
47
46
48
49
49
53
46
46
47
52
48
37
49
51
51
42
50
50
52
47
49
48
47
38
50
53
40
